app/controllers/social_networking/nudges_controller.rb

```diff
@@ -41,20 +39,14 @@ def notify
       when "email"
         send_notify_email(@nudge, message_body)
```

Comment:
Also, `message_body` is a method below. Is there a reason to pass it as a variable to the method?